robyn/openapi.py

```diff
@@ -239,13 +239,13 @@ def get_path_obj(
         query_params: Optional[TypedDict],
         request_body: Optional[TypedDict],
         return_annotation: Optional[TypedDict],
-    ) -> (str, dict):
+    ) -> tuple[str, dict]:
```

Comment:
we should use `typing.Tuple` here. Due to support of python3.8

Reply:
ok

Reply:
I've made that commit.
In the same method mypy complains about
```
        query_params: Optional[TypedDict],
        request_body: Optional[TypedDict],
        return_annotation: Optional[TypedDict],
```
with message `Variable "typing.TypedDict" is not valid as a type` I wasn't sure how to fix that. Their docs point us to https://mypy.readthedocs.io/en/stable/common_issues.html#variables-vs-type-aliases